src/main/java/Controllers/ReviewManager.java

```diff
@@ -0,0 +1,61 @@
+package Controllers;
+
+import Entities.Review;
+import UseCaseClasses.InformationSaver;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+public class ReviewManager {
+
+    private HashMap<String, Review> reviews;
+
+    public ReviewManager() {
+        reviews = new HashMap<>();
+    }
+
+    /**
+     *
+     * @param Location The location that the review is for.
+     * @param reviewBody The body of the review.
+     * @param rating The score out of 5 given by the reviewer.
+     * @return true if the review was successfully created and added
+     */
+
+    public boolean createReview(String Location, String reviewBody, Integer rating)
+            throws IOException, ExecutionException, InterruptedException, IllegalArgumentException{
+
+        InformationSaver saver = new InformationSaver();
+        boolean result = false;
+        if((rating > 5) || (rating < 0)){
+            result = false;
+        }
+        else {
+            Review review = new Review(Location, reviewBody, rating);
```

Comment:
I don't think we want to have ReviewManager using entities in the class, as it is a clean architecture violation. Refer to Rachel's changes in refactoring AccountManager.

Reply:
Specifically, create a class in the Use Case level that can perform these interactions with Entity classes and then call that method from ReviewManager